test/testastutils.cpp

```diff
@@ -81,7 +81,7 @@ class TestAstUtils : public TestFixture {
         Settings settings;
         Tokenizer tokenizer(&settings, this);
         std::istringstream istr(code);
-        tokenizer.tokenize(istr, "test.cpp");
+        ASSERT(tokenizer.tokenize(istr, "test.cpp")) false;
```

Comment:
before I merge I want that this is fixed.

Reply:
Ok, there is still a lot to do yet. You could also push this separately: https://github.com/danmar/cppcheck/pull/3501/commits/fa69d3c4fcc95f8331e0f80ae4f189ea759589f0

Reply:
I pushed that with f363d8948bab8894450349091b291e48ce478fe9